src/libDirectoryService/PoWProcessing.cpp

```diff
@@ -316,19 +312,17 @@ bool DirectoryService::ProcessPoWSubmissionFromPacket(
   bool result = POW::GetInstance().PoWVerify(
       blockNumber, difficultyLevel, headerHash, nonce, resultingHash, mixHash);
 
-  LOG_EPOCH(INFO, m_mediator.m_currentEpochNum,
-            "[POWSTAT] pow verify (microsec): " << r_timer_end(m_timespec));
+  LOG_GENERAL(INFO, "[POWSTAT] " << r_timer_end(m_timespec));
 
   if (result) {
     // Do another check on the state before accessing m_allPoWs
     // Accept slightly late entries as we need to multicast the DSBLOCK to
     // everyone if ((m_state != POW_SUBMISSION) && (m_state !=
     // DSBLOCK_CONSENSUS_PREP))
     if (!CheckState(VERIFYPOW)) {
-      LOG_EPOCH(INFO, m_mediator.m_currentEpochNum,
-                "Too late - current state is " << m_state);
+      LOG_GENERAL(INFO, "Too late. State = " << GetStateString());
     } else {
-      LOG_EPOCH(INFO, m_mediator.m_currentEpochNum, "POW verification passed");
+      LOG_GENERAL(INFO, "Verified OK");
```

Comment:
Log as "POW verified OK" will be more clear

Reply:
Won't be necessary, since you can already see from the function name that it is for checking PoW submission:

```
[INFO][15306][19-01-24T14:17:43.918][oWProcessing.cpp:325][ProcessPoWSubmission] Verified OK
```

I tried to reduce the logs as much as possible, avoid being too wordy and have too much info.

Reply:
BTW here is a fun fact, if we add something as small as "PoW ":

"PoW " = 4 bytes
4 bytes x 1800 submissions/epoch x 12 epochs/day x 365 days/year = 30MB per year!